src/test/java/com/revature/maadcars/services/UserServiceTest.java

```diff
@@ -0,0 +1,85 @@
+package com.revature.maadcars.services;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.revature.maadcars.controllers.UserController;
+import com.revature.maadcars.models.User;
+import com.revature.maadcars.repository.UserRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Matchers.any;
```

Comment:
Deprecated; there's an ArgumentMatchers.any() method inside the when() methods, take this import out to tell java to use that one instead.

Reply:
removed Matchers.any().
I just used ArgumentMatchers.anyString() instead